components/gitpod-protocol/src/messaging/error.ts

```diff
@@ -32,15 +32,18 @@ export namespace ErrorCodes {
     // 430 Repository not whitelisted (custom status code)
     export const REPOSITORY_NOT_WHITELISTED = 430;
 
+    // 450 Payment error
+    export const PAYMENT_ERROR = 450;
+
+    // 455 Invalid cost center (custom status code)
+    export const INVALID_COST_CENTER = 455;
```

Comment:
can we agree on a common prefix for everything payment related, please?

also see https://github.com/gitpod-io/gitpod/pull/11576/files#diff-3bb6d890c14ac42bf9860578a9f6a85ffd0121dbf980055baa99f8b7731c46daR45

and feel free to comment on that BTW!

alternative way to approach this, we could use just `PAYMENT_ERROR` (in similar cases) and encode specific conditions into the payload. it feels like we'll have the needs to extends for more error modes. wdyt?

Reply:
I don't see much value in aligning every payment-related error around a single prefix or error code range (apart from readability of the error codes list -- which we arguably don't read that often).

Personally, I'd prefer to leave this as is (unless you have a strong opinion towards a different error name or code), and you can use whichever convention seems best in your PR. We can always align later if you think it's valuable, but I suspect that aligning these is not important in the grand scheme of things. 😇